README.md

```diff
@@ -95,33 +95,26 @@ other manner of problems.  Caveat emptor.
 
 ## Getting Started
 
-First, clone this repository and [install rust](install-rust).
-
 To start your own project, you can use our `cargo risczero` tool to write the
-initial boilerplate and set up a standard file structure. First install the tool
+initial boilerplate and set up a standard directory structure.
+First install the tool:
```

Comment:
I'm agree with dropping the "clone this repo" sentence, but I think we should still mention installing Rust:
```suggestion
First, [install Rust][install-rust] if you don't already have it, then install the `cargo risczero` tool:
```